index.html

```diff
@@ -1,45 +1,63 @@
-          <img class="advantages__image" src="./images/HeadsetMicOutlined%201.png" alt="Support 24/7">
-          <h2 class="advantages__subtitle">Support 24/7</h2>
-        </div>
-      </li>
-      <li class="advantages__item">
-        <div class="advantages__communication">
-          <img class="advantages__image" src="./images/ForumOutlined%201.png" alt="Communicate directly">
-          <h2 class="advantages__subtitle">Communicate directly</h2>
-        </div>
-      </li>
-      <li class="advantages__item">
-        <div class="advantages__booking">
-          <img class="advantages__image" src="./images/DevicesOutlined%201.png" alt="Book online">
-          <h2 class="advantages__subtitle">Book online</h2>
-        </div>
-      </li>
-      <li class="advantages__item">
-        <div class="advantages__reviews">
-          <img class="advantages__image" src="./images/ThumbUpAltOutlined%201.png" alt="Real guest reviews">
-          <h2 class="advantages__subtitle">Real guest reviews</h2>
-        </div>
-      </li>
-    </ul>
-  </section>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="description" content="my first project on Front-end" />
+  <meta name="keywords" content="HTML, CSS, JS, React" />
+  <meta name="author" content="Maxim Petrovsky" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>TripHouse</title>
+  <link rel="stylesheet" href="./styles/styles.css">
+</head>
+<body>
+<div class="container advantages__container">
+  <h1 class="title advantages__title">What do we offer</h1>
+  <div class="advantages__highlight"></div>
+  <div class="row advantages__row">
+    <div class="col advantages__col">
+      <img class="advantages__image" src="./images/HeadsetMicOutlined%201.png" alt="Support 24/7">
+      <h2 class="subtitle advantages__subtitle">Support 24/7</h2>
+    </div>
+    <div class="col advantages__col">
+      <img class="advantages__image" src="./images/ForumOutlined%201.png" alt="Communicate directly">
+      <h2 class="subtitle advantages__subtitle">Communicate directly</h2>
+    </div>
+    <div class="col advantages__col">
+      <img class="advantages__image" src="./images/DevicesOutlined%201.png" alt="Book online">
+      <h2 class="subtitle advantages__subtitle">Book online</h2>
+    </div>
+    <div class="col advantages__col">
+      <img class="advantages__image" src="./images/ThumbUpAltOutlined%201.png" alt="Real guest reviews">
+      <h2 class="subtitle advantages__subtitle">Real guest reviews</h2>
+    </div>
+  </div>
+</div>
+<div class="container footer_container">
+  <div class="row footer_row">
+    <div class="col footer_col">
+      <p class="footer_col_title">About</p>
+      <p class="footer_link">How Triphouse works</p>
+      <p class="footer_link">Careers</p>
+      <p class="footer_link">Privacy</p>
+      <p class="footer_link">Terms</p>
+      <p class="footer_copyright">© 2020 Triphouse, Inc. All rights reserved</p>
```

Comment:
Why did you put it into the column? It looks separated in the design.
Also, you limited its width with the width of the column. Try to shrink the window to see.

Reply:
Would it be better to display these elements using a list?
Need to make the text in the column stretch in width when zooming out with a decrease in the distance between the column on the right?

Reply:
It doesn't make sense for me. But it is not a mistake to use a list here.
The text shouldn't be stretched. It is not clear what you gonna do. Look at copyright carefully - it is not on right place in your layout.